stable_baselines3/common/logger.py

```diff
@@ -190,25 +213,40 @@ def close(self) -> None:
         self.file.close()
 
 
-def valid_float_value(value: Any) -> bool:
-    """
-    Returns True if the value can be successfully cast into a float
+class TensorBoardOutputFormat(KVWriter):
+    def __init__(self, folder):
+        """
+        Dumps key/value pairs into TensorBoard's numeric format.
+        :param folder: (str) the folder to write the log to
+        """
+        self.writer = SummaryWriter(log_dir=folder)
 
-    :param value: (Any) the value to check
-    :return: (bool)
-    """
-    try:
-        float(value)
-        return True
-    except TypeError:
-        return False
+    def writekvs(self, kvs: Dict, excs: Dict, step: int) -> None:
+        for (key, val), (_, exc) in zip(sorted(kvs.items()), sorted(excs.items())):
+            if exc is not None:
+                if 'tensorboard' is exc or 'tensorboard' in exc:
```

Comment:
please use complete name (avoid `exc` and prefer `excluded`)
most of the logger code is from the old baselines... so not the best practices for naming...

Reply:
maybe a good time to rename `kvs` to something with more meaning

Reply:
Got it, will do a full renaming for logger then :)